content/documentation/getting-started/index.adoc

```diff
@@ -269,25 +269,27 @@ The above is actually the default when Kiali is installed on Kubernetes - so to
 
 === Login Options
 
-Kiali supports three different login options.
+Kiali supports four different login options.
 
 *login*: This option allows a user to login into Kiali using a username and password. This is the default option if using Kubernetes.
 
 *anonymous*: This option removes any login requirement. A user will not be presented the login page and will automatically have access to Kiali without having to present any credentials.
 
+*ldap*: This option allows a user to login into Kiali using a username and password that is authenticated via a backend LDAP server. This option requires that you configure additional settings in the Kiali CR under `auth.ldap`. Note that if you want to use this option, you cannot use the operator deploy script to configure Kiali for you - you must create the Kiali CR and either pass it to the deploy script via `--kiali-cr` or do not have the script deploy a Kiali CR (`--operator-install-kiali=false`) but instead deploy the Kiali CR directly into your cluster yourself.
```

Comment:
is it clear to users who read this how to configure the auth.ldap section? what are the fields and expected values? what is mandatory and what is optional? perhaps an example will help

Reply:
There is an example with the required settings below. I added a "see below" blurb here.